docs/getting_started/evaluate.md

```diff
@@ -0,0 +1,85 @@
+After the model has been trained, it can be used to predict the target output features on new data.
+
+We've created a small test dataset containing input features for 10 movie reviews that we can use for testing. Click this [link](https://github.com/ludwig-ai/ludwig-docs/raw/getting-started/docs/data/rotten_tomatoes_test.csv) to download the test dataset.
+
+Now, let's make some predictions on the test dataset (Note: if you are working in a notebook, you can use the model created in the training section):
+
+=== "CLI"
+
+    ``` sh
+    ludwig predict --model_path results/experiment_run/model --dataset rotten_tomatoes_test.csv
+    ```
+
+=== "Python"
+
+    ``` python
+    model = LudwigModel.load('results/experiment_run/model')
```

Comment:
```suggestion
    ``` python
    # This step can be skipped if you are working in a notebook, and you can simply 
    # re-use the model created in the training section.
    model = LudwigModel.load('results/experiment_run/model')
```